auth-filters/forgerock-authn-filter/forgerock-jaspi-modules/forgerock-jaspi-jwt-session-module/src/main/java/org/forgerock/jaspi/modules/session/jwt/AbstractJwtSessionModule.java

```diff
@@ -564,11 +573,11 @@ private String buildJwtString(JwtClaimsSet claimsSet, Key publicKey) {
      * @param now The Date at which the cookie was created.
      * @param exp The expiry Date of the cookie.
      */
-    private int getCookieMaxAge(Date now, Date exp) {
+    private Integer getCookieMaxAge(Date now, Date exp) {
```

Comment:
Javadoc of this method is not valid anymore. Could you please reword it?

Reply:
Changed back to previous semantic (see comment bellow).